src/pages/host/getHostEventString.tsx

```diff
@@ -225,7 +224,7 @@ export const getHostEventString = (
           Current running task cleared (was:
           <StyledLink
             data-cy="host-running-task-cleared-link"
-            href={`${getUiUrl()}/task${data.taskId}/${data.execution}`}
+            href={`/task/${data.taskId}/${data.execution}`}
```

Comment:
Could you use the utility function i mentioned above for these use cases. You will need to modify it to also take in execution as an optional paramater

Reply:
I actually won't use execution because it's null